### ISSUE TITLE
Remote API not working anymore 

### ISSUE THREAD

**Issue by irioca** [user2684/imou_life#118]
Hello,
integration was working fine till some days ago, then stopped.
Tried to delete the configuration and create from scratch but now still get the error: Remote API Error.
Please support

**Comment by wingmanx** [comment]
It stopped working almost a week ago.
This address give a 404 error:
https://openapi.easy4ip.com/openapi

Here in this link below we can see that the address shown for the console is https://openapi.easy4ip.com/consoleNew/xx
https://open.imoulife.com/book/readme/create.html

But if you open the imou developer the link shows:
https://open.imoulife.com/consoleNew/xx

Perhaps something has changed in the API Base URL?

**Comment by tomadan1968** [comment]
The same problem for me...

**Comment by s2splus** [comment]
Hello,

Same for me

**Comment by user2684** [comment]
Hi guys, I see a lot of weird things happening on the Imou side these days, like #117. For me everything is working fine for example but from time to time I get API errors which solves by themselves after a few days. Unfortunately all of these are remote errors which we cannot troubleshoot by any means and Imou is not helping either because they change from time to time their APIs without updating the doc (actually is no more updated since 2021 now) :-( 
Somebody like #101 changed the base URL and it fixed maybe a similar problem but unfortunately there is no consistency in these fixes...

**Comment by jmbenevise** [comment]
Have a look at the imoulife website: https://open.imoulife.com/book/guide/haDev.html

**Comment by jeroen-groeneweg** [comment]
I can confirm that transfering $1 into you imou account will "solve" the issue. Even though my logs didn't mention API request limit I was being blocked due to max API request limit. See: https://github.com/user2684/imou_life/wiki/Imou-API-usage-restrictions 

I will now configure my camera as an ONVIF device to bypass the API. Again, making sure you have a positive balance will also solve it (for now).

**Comment by wingmanx** [comment]
And ... Just like that. Everything's working.
Haven't changed anything or transferred any amount.

Perhaps is that max API request limit?

**Comment by user2684** [comment]
@jmbenevise this is interesting, looks like Imou published their own official home assistant integration. Have you tried it already? They definitely have better knowledge of their own API then myself, if it works well, I can consider deprecating this integration. Any feedback please let me know. Thanks

**Comment by Ices-Eyes** [comment]
> @jmbenevise this is interesting, looks like Imou published their own official home assistant integration. Have you tried it already? They definitely have better knowledge of their own API then myself, if it works well, I can consider deprecating this integration. Any feedback please let me know. Thanks

Hi... Just tried the imou integration, really poor on features... They seems to support dual camera video stream, but the available features are really low... I.e., these are the function for a Ranger Dual Camera (same features for a Cruiser Dual)
![image](https://github.com/user-attachments/assets/06a682d4-4627-430f-ad8f-e4b03020bd0d)

On the other hand, some more features are available for a Cell 3C:
![image](https://github.com/user-attachments/assets/86ff930a-0bfd-4357-b478-d9f20053aa50)


Hope you can take some info from the official integration, but do not deprecate this!

**Comment by sturlahallas** [comment]
I think they actually pulled their own integration. I didn't work that well, and it sucked my Imou account dry really fast...

**Comment by wingmanx** [comment]
I've finally solved all the problems I had with imou life cams. It's an easy trick, and solved all my problems!

Besides the constant API problems I've described above, the latest problem I've encountered is the impossibility to have the cam working only in local network.
If you block the internet connection of the cam, to work with it only locally through RTSP in home assistant, the camera enters in an endless bootloop until it can "phone home", rendering the camera unusable.
After contacting imoulife about this strange camera behaviour, this is their answer:
"Thanks for contacting IMOU. I have checked your problem and would like to tell you that Cruiser SE+ requires constant internet connection".

So, problem solved!!! Bought Tapo Cams, and I'm selling 7 imou life cameras.

I WILL NEVER EVER BUY AGAIN THIS PIECE OF SHXX OF IMOU LIFE CAMS.

**Comment by user2684** [comment]
Hi, just to confirm there is an official integration out there, for this I have recently decided to stop actively developing this unofficial integration. For more information, have a look at https://github.com/user2684/imou_life/wiki/Status-of-the-integration
And correct, from what I can understand as of today there is no way to configure in the official integration the polling interval so can easily reach the API limit unless not recharged

**Comment by Ices-Eyes** [comment]
It's a pity this integration goes off, the official one is really crap, and working really bad, other than consuming all the free credit in a couple of days... BTW thanks for your effort and time :)

**Comment by Jonsonmon** [comment]
> I've finally solved all the problems I had with imou life cams. It's an easy trick, and solved all my problems!
> 
> Besides the constant API problems I've described above, the latest problem I've encountered is the impossibility to have the cam working only in local network. If you block the internet connection of the cam, to work with it only locally through RTSP in home assistant, the camera enters in an endless bootloop until it can "phone home", rendering the camera unusable. After contacting imoulife about this strange camera behaviour, this is their answer: "Thanks for contacting IMOU. I have checked your problem and would like to tell you that Cruiser SE+ requires constant internet connection".
> 
> So, problem solved!!! Bought Tapo Cams, and I'm selling 7 imou life cameras.
> 
> I WILL NEVER EVER BUY AGAIN THIS PIECE OF SHXX OF IMOU LIFE CAMS.

You are right. After 2 days with WAN blocked, the camera ONVIF is not working either.
Any solution for that?

**Comment by wingmanx** [comment]
> > I've finally solved all the problems I had with imou life cams. It's an easy trick, and solved all my problems!
> > Besides the constant API problems I've described above, the latest problem I've encountered is the impossibility to have the cam working only in local network. If you block the internet connection of the cam, to work with it only locally through RTSP in home assistant, the camera enters in an endless bootloop until it can "phone home", rendering the camera unusable. After contacting imoulife about this strange camera behaviour, this is their answer: "Thanks for contacting IMOU. I have checked your problem and would like to tell you that Cruiser SE+ requires constant internet connection".
> > So, problem solved!!! Bought Tapo Cams, and I'm selling 7 imou life cameras.
> > I WILL NEVER EVER BUY AGAIN THIS PIECE OF SHXX OF IMOU LIFE CAMS.
> 
> You are right. After 2 days with WAN blocked, the camera ONVIF is not working either. Any solution for that?

Yes, solution is in my message ...
"problem solved!!! Bought Tapo Cams"

**Comment by Jonsonmon** [comment]
> > > I've finally solved all the problems I had with imou life cams. It's an easy trick, and solved all my problems!
> > > Besides the constant API problems I've described above, the latest problem I've encountered is the impossibility to have the cam working only in local network. If you block the internet connection of the cam, to work with it only locally through RTSP in home assistant, the camera enters in an endless bootloop until it can "phone home", rendering the camera unusable. After contacting imoulife about this strange camera behaviour, this is their answer: "Thanks for contacting IMOU. I have checked your problem and would like to tell you that Cruiser SE+ requires constant internet connection".
> > > So, problem solved!!! Bought Tapo Cams, and I'm selling 7 imou life cameras.
> > > I WILL NEVER EVER BUY AGAIN THIS PIECE OF SHXX OF IMOU LIFE CAMS.
> > 
> > 
> > You are right. After 2 days with WAN blocked, the camera ONVIF is not working either. Any solution for that?
> 
> Yes, solution is in my message ... "problem solved!!! Bought Tapo Cams"

I will try to enable only 123 port (NTP)
Maybe that't what makes the problems
Thanks

**Comment by Jonsonmon** [comment]
> > > I've finally solved all the problems I had with imou life cams. It's an easy trick, and solved all my problems!
> > > Besides the constant API problems I've described above, the latest problem I've encountered is the impossibility to have the cam working only in local network. If you block the internet connection of the cam, to work with it only locally through RTSP in home assistant, the camera enters in an endless bootloop until it can "phone home", rendering the camera unusable. After contacting imoulife about this strange camera behaviour, this is their answer: "Thanks for contacting IMOU. I have checked your problem and would like to tell you that Cruiser SE+ requires constant internet connection".
> > > So, problem solved!!! Bought Tapo Cams, and I'm selling 7 imou life cameras.
> > > I WILL NEVER EVER BUY AGAIN THIS PIECE OF SHXX OF IMOU LIFE CAMS.
> > 
> > 
> > You are right. After 2 days with WAN blocked, the camera ONVIF is not working either. Any solution for that?
> 
> Yes, solution is in my message ... "problem solved!!! Bought Tapo Cams"

If you still have imou cams. I can help you.

**Comment by wingmanx** [comment]
I've sold them all, now I only have Tapo Cams, working flawlessly for 8 months now.
But thanks anyway.
Imou cams have good hardware quality, but not so good in software for those who want more control of the device.
A simple thing as prevent "phone home" was replied by Imou itself, saying that is not possible, so, at that moment, I quit Imou.
